---
layout: page
title: Papers
---

More info on my [Google Scholar profile](http://scholar.google.com/citations?user=HI-I6C0AAAAJ&hl=en&oi=ao)


### Preprints

* Patil P, Rosenblum MA, Leek JT (2015) _Measuring the contribution of genomic predictors to improving estimator precision in randomized trials_ bioRxiv doi:10.1101/018168 [[link]](http://biorxiv.org/content/early/2015/04/16/018168) [[analysis code]](https://github.com/leekgroup/genesigprecision)

* Collado Torres L, Jaffe AE, and Leek JT (2015) _regionReport: Interactive reports for region-based analyses_ bioRxiv doi:10.1101/016659 [[link]](http://biorxiv.org/content/early/2015/03/17/016659) [[software]](http://master.bioconductor.org/packages/release/bioc/html/regionReport.html)

* Collado Torres L, Frazee AC, Love MI, Irizarry RA, Jaffe AE, Leek JT _derfinder: Software for annotation-agnostic RNA-seq differential expression analysis_ bioRxiv doi:10.1101/015370 [[link]](http://biorxiv.org/content/early/2015/02/19/015370) [[software]](http://www.bioconductor.org/packages/release/bioc/html/derfinder.html)

* Frazee AC, Jaffe AE, Langmead B, Leek JT (2014) _Polyester: simulating RNA-seq dataets with differential expression_ bioRxiv doi: 10.1101/006015 [[link]](http://biorxiv.org/content/early/2014/06/06/006015) [[software devel]](https://github.com/alyssafrazee/polyester)[[software release]](http://www.bioconductor.org/packages/release/bioc/html/polyester.html) [[analysis code]](https://github.com/alyssafrazee/polyester_code)



### 2015

* Patil P, Bachant-Winner PO, Haibe-Kains B, Leek JT (2014) _Avoiding test set bias with rank-based predictors_ Bioinformatics doi: 10.1093/bioinformatics/btv157 [[link]](http://bioinformatics.oxfordjournals.org/content/early/2015/03/18/bioinformatics.btv157.full.pdf?keytype=ref&ijkey=loVpUJfJxG2QjoE) [[preprint]](http://biorxiv.org/content/early/2014/06/06/005983) [[analysis code]](https://github.com/prpatil/testsetbias)


* Frazee AC, Pertea G, Jaffe AE, Langmead B, Salzberg SL, Leek JT (2015) _Ballgown bridges the gap between transcriptome assembly and expression analysis_ Nature Biotechnology 33, 243–246 (2015) [[link]](http://www.nature.com/nbt/journal/v33/n3/full/nbt.3172.html) [[supplement]](https://github.com/jtleek/jtleek.github.io/blob/master/papers/ballgown_supp.pdf?raw=true) [[preprint]](http://biorxiv.org/content/early/2014/03/30/003665) [[software devel]](https://github.com/alyssafrazee/ballgown) [[software release]](http://www.bioconductor.org/packages/release/bioc/html/ballgown.html) [[analysis code]](https://github.com/alyssafrazee/ballgown_code) [[geuvadis data]](https://www.dropbox.com/s/kp5th9hgkq8ckom/geuvadisbg.rda)



* Leek JT and Peng RD (2015) _What is the question?_ Science DOI: 10.1126/science.aaa6146 [[link]](http://www.sciencemag.org/cgi/rapidpdf/science.aaa6146?ijkey=aB/od5cvz9dB6&keytype=ref&siteid=sci)

* Leek JT and Peng RD (2015) _Opinion: Reproducible research can still be wrong: Adopting a prevention approach_ PNAS 112 (6) 1645-1645 [[link]](http://www.pnas.org/content/112/6/1645.full) [[preprint]](http://arxiv.org/abs/1502.03169)


### 2014


* Jaffe AE, Shin J, Collado-Torres L, Leek JT, Tao R, Li C, Gao Y, Jia Y, Maher BJ, Hyde TM, Kleinman JE, Weinberger DR (2014) _Developmental regulation of human cortex transcription and its clinical relevance at single base resolution_ Nature Neuroscience doi:10.1038/nn.3898 [[link]](http://www.nature.com/neuro/journal/vaop/ncurrent/full/nn.3898.html)


* Fisher A, Anderson GB, Leek JT (2014) _A randomized trial in a massive online open course shows people don't know what a statistically significant relationship looks like, but they can learn_ PeerJ 2:e589 DOI 10.7717/peerj.589 [[paper]](https://peerj.com/articles/589/) [[preprint]](http://arxiv.org/abs/1404.5358) [[p-value shiny app]](http://glimmer.rstudio.com/afisher/EDA/)


* Leek JT (2014) _svaseq: removing batch effects and other unwanted noise from sequencing data_ Nucl. Acids Res. doi: 10.1093/nar/gku864 [[paper]](http://nar.oxfordjournals.org/content/early/2014/10/07/nar.gku864.full?keytype=ref&ijkey=Rq9NTea3ZraBKMQ) [[preprint]](http://biorxiv.org/content/early/2014/06/25/006585) [[software]](http://www.bioconductor.org/packages/devel/bioc/html/sva.html) [[analysis code]](https://github.com/jtleek/svaseq)

* Parker HS, Corrada Bravo H, Leek JT. (2014) _Removing batch effects for prediction problems with frozen surrogate variable analysis._ PeerJ 2:e561 DOI 10.7717/peerj.561 [[paper]](http://dx.doi.org/10.7717/peerj.561) [[preprint]](http://arxiv.org/abs/1301.3947) [[software]](http://www.bioconductor.org/packages/devel/bioc/html/sva.html)

* Parker HS, Leek JT, Favorov AV, Considine M, Xia X, Chavan S, Chung CH, Fertig EJ (2014) _Preserving biological heterogeneity with a permuted surrogate variable analysis for genomics batch correction_ [[link]](http://bioinformatics.oxfordjournals.org/content/early/2014/06/06/bioinformatics.btu375.long) [[analysis code]](https://sourceforge.net/projects/psva) [[software]](http://www.bioconductor.org/packages/devel/bioc/html/sva.html)

* Hayes LN, Severance EG, Leek JT, Gressitt KL, Rohleder C, Coughlin JM, Leweke FM, Yolken RH, Sawa A. (2014) _Inflammatory molecular signature associated with infectious agents in psychosis_. Scizhophrenia Bulletin doi: 10.1093/schbul/sbu052 [[link]](http://schizophreniabulletin.oxfordjournals.org/content/early/2014/04/16/schbul.sbu052.full)

* Frazee AC, Sabunciyan S, Hansen KD, Irizarry RA, Leek JT (2014) _Differential expression analysis of RNA-seq data at single-base resolution_ Biostatistics 10.1093/biostatistics/kxt053 [[link]](http://biostatistics.oxfordjournals.org/content/early/2014/01/06/biostatistics.kxt053.short)[[software]](https://github.com/alyssafrazee/derfinder) [[newer software]](https://github.com/lcolladotor/derfinder)

* Jager LR, Leek JT (2014) _An estimate of the science-wise false discovery rate and application to the top medical literature_ Biostatistics 15(1) 1-12 [[link]](http://biostatistics.oxfordjournals.org/content/15/1/1.short)[[blog post]](http://simplystatistics.org/2013/09/25/is-most-science-false-the-titans-weigh-in/) [[code]](https://github.com/jtleek/swfdr) [data](https://github.com/jtleek/swfdr/blob/master/pvalueData.rda) [[preprint]](http://arxiv.org/pdf/1301.3718.pdf)

* Jager LR, Leek JT (2014) _Rejoinder: An estimate of the science-wise false discovery rate and application to the top medical literature_ Biostatistics 15(1) 39-45 [[link]](http://biostatistics.oxfordjournals.org/content/15/1/39)


### 2013

* Jaffe AE, Storey JD, Ji H, Leek JT (2013) _Gene set bagging for estimating the probability a statistically significant result will replicate_ BMC Bioinformatics 14 (1), 360 [[link]](http://www.biomedcentral.com/1471-2105/14/360/) [code](https://github.com/andrewejaffe/GeneSetBagging)

* Marchionni L, Afsari B, Geman D, Leek JT (2013) _A simple and reproducible breast cancer prognostic test_ BMC Genomics 14 (1) 336[[link]](http://www.biomedcentral.com/1471-2164/14/336?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+Bmc%2FGenomics%2FLatestArticles+(BMC+Genomics+-+Latest+articles)) [data and code](http://astor.som.jhmi.edu/~marchion//breastTSP.html)

* Leek JT, Salzberg SL (2013) _Sequestration: inadvertently killing biomedical research to score political points_ Genome Biology 14 (3), 109 [[link]](http://www.biomedcentral.com/content/pdf/gb-2013-14-3-109.pdf)

* Pirooznia M, Seifuddin F, Goes FS, Leek JT, Zandi PP (2013) _SVAw - a web-based application tool for automated surrogate variable analysis of gene expression studies_ Source Code for Biology and Medicine 2013, 8:8 [[link]](http://www.biomedcentral.com/content/pdf/1751-0473-8-8.pdf) [[web app]](http://psychiatry.igm.jhmi.edu/sva/)

* Boca SM, Corrada Bravo H, Caffo B, Leek JT, and Parmigiani G. (2013) _A decision-theory approach to interpretable set analysis for high-dimensional data._ Biometrics 69(3) 614-623. [[pdf]](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3927844/) [[preprint]](http://biostats.bepress.com/jhubiostat/paper211/)

### 2012 

* Corrada Bravo H, Pihur V, McCall M, Irizarry RA, Leek JT (2012). _Gene expression anti-profiles as a basis for accurate universal cancer signatures_ BMC Bioinformatics 13:272 [[link]](http://www.biomedcentral.com/1471-2105/13/272) [data](http://bioconductor.org/packages/release/data/experiment/html/antiProfilesData.html) [[code]](http://bioconductor.org/packages/release/bioc/html/antiProfiles.html)

* Leek JT, Rasgon JL, Taub MA. (2012) _A statistical approach to selecting and confirming validation targets in -omics experiments_ BMC Bioinformatics 13:150 [[link]](http://www.biomedcentral.com/1471-2105/13/150) [[software]](http://biostat.jhsph.edu/~jleek/validate/)

* Leek JT, Peng RD, and Anderson RR. (2012) _Personalized medicine: Keep a way open for tailored treatments_ Nature 484: 318. [[submitted version]](https://docs.google.com/document/d/17FHbaabQGAFqlZcaoz7OJXBRLv4AZtlgiva0LLMIDNk/edit) [[link]](http://www.nature.com/nature/journal/v484/n7394/full/484318a.html) [[on the blog]](http://simplystatistics.org/2012/03/20/laws-of-nature-and-the-law-of-patents-supreme-court/)

* Parker HS and Leek JT. (2012) _The practical effect of batch on genomic prediction_ SAGMB 11: 3 [[preprint]](http://biostat.jhsph.edu/~jleek/papers/practicalbatch_sagmb.pdf) [[link]](http://www.degruyter.com/view/j/sagmb.2012.11.issue-3/1544-6115.1766/1544-6115.1766.xml) [[data]](http://biostat.jhsph.edu/~hiparker/PracticalBatch/)

* Jaffe AE, Murakami P, Lee H, Leek JT, Fallin MD, Feinberg AP, Irizarry RA. (2012) _Bump hunting to identify differentially methylated regions in epigenetic epidemiology studies_ International Journal of Epidemiology 41:200-209. [[link]](http://ije.oxfordjournals.org/content/41/1/200.short)

* Leek JT, Johnson WE, Parker HS, Jaffe AE, and Storey JD. (2012) _The sva package for removing batch effects and other unwanted variation in high-throughput experiments._ Bioinformatics DOI:10.1093/bioinformatics/bts034 [[link]](http://bioinformatics.oxfordjournals.org/content/28/6/882.long) [[software]](http://bioconductor.org/packages/release/bioc/html/sva.html)

* Jaffe AE, Feinberg AP, Irizarry IR, and Leek JT.(2011) _Significance analysis and statistical dissection of variably methylated regions_ Biostatistics 13: 166-178 [[link]](http://biostatistics.oxfordjournals.org/content/13/1/166.short) [[pdf]](http://biostat.jhsph.edu/~jleek/papers/jaffe-vmrs.pdf) [[code]](http://www.biostat.jhsph.edu/~ajaffe/code/vmrFinder.R)

### 2011 

* Frazee AC, Langmead B, and Leek JT. (2011) _ReCount: A multi-experiment resource of analysis-ready RNA-seq gene count datasets_ BMC Bioinformatics 12:449 [[link]](http://www.biomedcentral.com/1471-2105/12/449) [[data]](http://bowtie-bio.sourceforge.net/recount/)

* Leek JT, Taub MA, Pineda FJ. (2011) _Cooperation between referees and authors increases peer review accuracy_ PLoS One 6:e26895 [[link]](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0026895) [[data]](http://biostat.jhsph.edu/~jleek/peerreview/)

* Colantuoni C, Lipska BK, Hyde TM, Tao R, Leek JT , Colantuoni EA, Elkahlhoun AG, Herman MM, Weinberger DR, Kleinman JE.(2011) _Temporal dynamics and genetic control of transcription in the human prefrontal cortex_ Nature 478: 519-523. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/carlosbrain.pdf) [[data]](http://braincloud.jhmi.edu/)

* Desai KH, Tan CS, Leek JT , Maier RV, Tompkins RG, Storey JD, and IHGRI.(2011) _Dissecting inflammatory complications in critically injured patients by within-patient gene expression changes: A longitudinal clinical genomics study_ PLoS Medicine 8:e1001093 [[link]](http://www.plosmedicine.org/article/info%3Adoi%2F10.1371%2Fjournal.pmed.1001093) [[data]](http://genomine.org/trauma/)

* Hansen KD, Wu Z, Irizarry RI, and Leek JT. (2011) _Sequencing technology does not eliminate biological variability._ Nature Biotechnology, 29:572â€“573 [[pdf]](http://biostat.jhsph.edu/~jleek/papers/seqvar.pdf)

* Leek JT, Storey JD (2011) _The joint null criterion for multiple hypothesis tests._ Statistical Applications in Genetics and Molecular Biology 10:28 [[pdf]](http://biostat.jhsph.edu/~jleek/papers/jointnull.pdf) [[code]](http://www.bioconductor.org/packages/devel/bioc/html/dks.html)

* Woo S, Leek JT, Storey JD (2011) _A computationally efficient modular optimal discovery procedure._ Bioinformatics 27:509-515 [[link]](http://bioinformatics.oxfordjournals.org/content/27/4/509) [[pdf]](http://biostat.jhsph.edu/~jleek/papers/odp-computation.pdf)

* Leek JT. (2011) _Asymptotic conditional singular value decomposition for high-dimensional genomic data._ Biometrics 67:344-352 [[pdf]](http://biostat.jhsph.edu/~jleek/papers/acsvd.pdf) [[code]](http://biostat.jhsph.edu/~jleek/code/dimensionfunc.R)

### 2010 

* Leek JT , Scharpf RB, Corrada Bravo H, Simcha D, Langmead B, Johnson WE, Geman D, Baggerly K, Irizarry RA (2010), _Tackling the widespread and critical impact of batch effects in high-throughput data._ Nature Reviews Genetics 11:733-739 [[link]](http://www.nature.com/nrg/journal/v11/n10/abs/nrg2825.html) [[pdf]](http://biostat.jhsph.edu/~jleek/papers/batch-review.pdf) [[data]](http://rafalab.jhsph.edu/batch/)

* Langmead B, Hansen KD, Leek JT (2010), _Cloud-scale RNA-sequencing differential expression analysis with Myrna._ Genome Biology 11:R83. [[link]](http://genomebiology.com/2010/11/8/R83/abstract) [[pdf]](http://biostat.jhsph.edu/~jleek/papers/myrna.pdf) [[code]](http://bowtie-bio.sourceforge.net/myrna/index.shtml)

* Lu R, Markowetz F, Unwin RD, Leek JT , Airoldi EM, MacArthur BD, Lachmann A, Rozov R, Ma'ayan A, Boyer LA, Troyanskaya OG, Whetton AD, and Lemischka IR. (2010) _Systems-level dynamic analyses of fate change in murine embryonic stem cells._ Nature 462: 358:362. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/systems.pdf)

### 2009 and Earlier 

* Leek JT. (2009) _The tspair package for finding top scoring pair classifiers in R._ Bioinformatics 25:1203-1204. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/tspair.pdf) [[software]](http://www.bioconductor.org/packages/2.5/bioc/html/tspair.html)

* Leek JT and Storey JD. (2008) _A general framework for multiple testing dependence._ Proceedings of the National Academy of Sciences , 105: 18718-18723. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/framework.pdf) [[software]](http://bioconductor.org/packages/release/bioc/html/sva.html)

* Leek JT and Storey JD. (2007) _Capturing heterogeneity in gene expression studies by `Surrogate Variable Analysis'._ PLoS Genetics, 3: e161. [[link]](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.0030161) [[software]](http://bioconductor.org/packages/release/bioc/html/sva.html)

* Akey JM, Biswas S, Leek JT, and Storey JD. (2007) _On the design and analysis of expression studies in human populations._ Nature Genetics, 39: 807-808. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/design.pdf)

* Storey JD, Dai JY, and Leek JT. (2006), _The optimal discovery procedure for large-scale significance testing, with applications to comparative microarray experiments._ Biostatistics, 8: 414-432. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/odp.pdf) [[software]](http://www.genomine.org/edge/)

* Leek JT, Monsen E, Dabney AR, and Storey JD. (2006), _EDGE: Extraction and analysis of differential gene expression._ Bioinformatics, 22: 507-508. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/edge.pdf) [[software]](http://www.genomine.org/edge/)

* Storey JD, Xiao W, Leek JT, Tompkins RG, and Davis RW. (2005), _Significance analysis of timecourse microarray experiments._ Proceedings of the National Academy of Sciences, 102: 12837-12842. [[pdf]](http://biostat.jhsph.edu/~jleek/papers/time.pdf) [[software]](http://www.genomine.org/edge/)
