<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Papers &middot; Leek
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/hyde.css">
  <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700|Abril+Fatface">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-144-precomposed.png">
                                 <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body>

    <div class="sidebar">
  <div class="container">
    <div class="sidebar-about">
      <h1>Leek</h1>
      <p class="lead">Find me online <a href="http://twitter.com/jtleek">@jtleek</a>, <a href="http://twitter.com/simplystats">@simplystats</a>, <a href="http://simplystatistics.org">Simply Statistics</a>, and <a href="https://github.com/jtleek">Github</a>.</p>
    </div>

    <ul class="sidebar-nav">
      <li class="sidebar-nav-item">
        <a href="/">Home</a>
      </li>

      

      
      
        
          
        
      
        
          
            <li class="sidebar-nav-item">
              <a href="/alumni/">Alumni</a>
            </li>
          
        
      
        
      
        
          
        
      
        
          
            <li class="sidebar-nav-item active">
              <a href="/papers/">Papers</a>
            </li>
          
        
      
        
          
            <li class="sidebar-nav-item">
              <a href="/people/">People</a>
            </li>
          
        
      
        
          
            <li class="sidebar-nav-item">
              <a href="/software/">Software</a>
            </li>
          
        
      
        
          
            <li class="sidebar-nav-item">
              <a href="/teaching/">Teaching</a>
            </li>
          
        
      

    </ul>

  </div>
</div>


    <div class="content container">
      <div class="page">
  <h1 class="page-title">Papers</h1>
  <p>More info on my <a href="http://scholar.google.com/citations?user=HI-I6C0AAAAJ&amp;hl=en&amp;oi=ao">Google Scholar profile</a></p>

<h2>Preprints</h2>

<ul>
<li>Frazee AC, Pertea G, Jaffe AE, Langmead B, Salzberg SL, Leek JT (2014) <em>Flexible isoform-level differential expression analysis with Ballgown</em> bioRxiv doi: 10.1101/003665 <a href="http://biorxiv.org/content/early/2014/03/30/003665">[link]</a> <a href="https://github.com/alyssafrazee/ballgown">[software]</a> <a href="https://github.com/alyssafrazee/ballgown_code">[analysis code]</a> <a href="https://www.dropbox.com/s/kp5th9hgkq8ckom/geuvadisbg.rda">[geuvadis data]</a></li>
<li>Fisher A, Anderson GB, Leek JT (2014) <em>A randomized trial in a massive online open course shows people don&#39;t know what a statistically significant relationship looks like, but they can learn</em>  arXiv:1404.5358 <a href="http://arxiv.org/abs/1404.5358">[link]</a> <a href="http://glimmer.rstudio.com/afisher/EDA/">[p-value shiny app]</a></li>
<li>Parker HS, Bravo HC, Leek JT (2013) <em>Removing batch effects for prediction problems with frozen surrogate variable analysis</em>  arXiv:1301.3947 <a href="http://arxiv.org/abs/1301.3947">[link]</a> <a href="http://bioconductor.org/packages/release/bioc/html/sva.html">[code]</a></li>
</ul>

<h2>2014</h2>

<ul>
<li>Hayes LN, Severance EG, Leek JT, Gressitt KL, Rohleder C, Coughlin JM, Leweke FM, Yolken RH, Sawa A. (2014) <em>Inflammatory molecular signature associated with infectious agents in psychosis</em>. Scizhophrenia Bulletin doi: 10.1093/schbul/sbu052 <a href="http://schizophreniabulletin.oxfordjournals.org/content/early/2014/04/16/schbul.sbu052.full">[link]</a></li>
<li>Frazee AC, Sabunciyan S, Hansen KD, Irizarry RA, Leek JT (2014) <em>Differential expression analysis of RNA-seq data at single-base resolution</em> Biostatistics 10.1093/biostatistics/kxt053 <a href="http://biostatistics.oxfordjournals.org/content/early/2014/01/06/biostatistics.kxt053.short">[link]</a> <a href="https://github.com/alyssafrazee/derfinder">[software]</a> <a href="https://github.com/lcolladotor/derfinder">[newer software]</a></li>
<li>Jager LR, Leek JT (2014) <em>An estimate of the science-wise false discovery rate and application to the top medical literature</em> Biostatistics 15(1) 1-12 <a href="http://biostatistics.oxfordjournals.org/content/15/1/1.short">[link]</a><a href="http://simplystatistics.org/2013/09/25/is-most-science-false-the-titans-weigh-in/">[blog post]</a> <a href="https://github.com/jtleek/swfdr">[code]</a>  <a href="https://github.com/jtleek/swfdr/blob/master/pvalueData.rda">[data]</a> <a href="http://arxiv.org/pdf/1301.3718.pdf">[preprint]</a></li>
<li>Jager LR, Leek JT (2014) <em>Rejoinder: An estimate of the science-wise false discovery rate and application to the top medical literature</em> Biostatistics 15(1) 39-45 <a href="http://biostatistics.oxfordjournals.org/content/15/1/39">[link]</a></li>
</ul>

<h2>2013</h2>

<ul>
<li>Jaffe AE, Storey JD, Ji H, Leek JT (2013) <em>Gene set bagging for estimating the probability a statistically significant result will replicate</em> BMC Bioinformatics 14 (1), 360 <a href="http://www.biomedcentral.com/1471-2105/14/360/">[link]</a> <a href="https://github.com/andrewejaffe/GeneSetBagging">[code]</a></li>
<li>Marchionni L, Afsari B, Geman D, Leek JT (2013) <em>A simple and reproducible breast cancer prognostic test</em> BMC Genomics 14 (1) 336<a href="http://www.biomedcentral.com/1471-2164/14/336?utm_source=feedburner&amp;utm_medium=feed&amp;utm_campaign=Feed%3A+Bmc%2FGenomics%2FLatestArticles+(BMC+Genomics+-+Latest+articles">[link]</a> <a href="http://astor.som.jhmi.edu/%7Emarchion/breastTSP.html">[data and code]</a></li>
<li>Leek JT, Salzberg SL (2013) <em>Sequestration: inadvertently killing biomedical research to score political points</em> Genome Biology 14 (3), 109 <a href="http://www.biomedcentral.com/content/pdf/gb-2013-14-3-109.pdf">[link]</a></li>
<li>Pirooznia M, Seifuddin F, Goes FS, Leek JT, Zandi PP (2013) <em>SVAw - a web-based application tool for automated surrogate variable analysis of gene expression studies</em> Source Code for Biology and Medicine 2013, 8:8 <a href="http://www.biomedcentral.com/content/pdf/1751-0473-8-8.pdf">[link]</a> <a href="http://psychiatry.igm.jhmi.edu/sva/">[web app]</a></li>
</ul>

<h3>2012</h3>

<ul>
<li>Corrada Bravo H, Pihur V, McCall M, Irizarry RA, Leek JT (2012). <em>Gene expression anti-profiles as a basis for accurate universal cancer signatures</em> BMC Bioinformatics 13:272 <a href="http://www.biomedcentral.com/1471-2105/13/272">[link]</a> <a href="http://bioconductor.org/packages/release/data/experiment/html/antiProfilesData.html">[data]</a> <a href="http://bioconductor.org/packages/release/bioc/html/antiProfiles.html">[code]</a></li>
<li>Leek JT, Rasgon JL, Taub MA. (2012) <em>A statistical approach to selecting and confirming validation targets in -omics experiments</em> BMC Bioinformatics 13:150 <a href="http://www.biomedcentral.com/1471-2105/13/150">[link]</a> <a href="http://biostat.jhsph.edu/%7Ejleek/validate/">[software]</a></li>
<li>Leek JT, Peng RD, and Anderson RR. (2012) &quot;Personalized medicine: Keep a way open for tailored treatments&quot; Nature 484: 318. <a href="https://docs.google.com/document/d/17FHbaabQGAFqlZcaoz7OJXBRLv4AZtlgiva0LLMIDNk/edit">[submitted version]</a> <a href="http://www.nature.com/nature/journal/v484/n7394/full/484318a.html">[link]</a> <a href="http://simplystatistics.org/2012/03/20/laws-of-nature-and-the-law-of-patents-supreme-court/">[on the blog]</a></li>
<li>Parker HS and Leek JT. (2012) &quot;The practical effect of batch on genomic prediction&quot; SAGMB 11: 3 <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/practicalbatch_sagmb.pdf">[preprint]</a> <a href="http://www.degruyter.com/view/j/sagmb.2012.11.issue-3/1544-6115.1766/1544-6115.1766.xml">[link]</a> <a href="http://biostat.jhsph.edu/%7Ehiparker/PracticalBatch/">[data]</a></li>
<li>Jaffe AE, Murakami P, Lee H, Leek JT, Fallin MD, Feinberg AP, Irizarry RA. (2012) &quot;Bump hunting to identify differentially methylated regions in epigenetic epidemiology studies&quot; International Journal of Epidemiology 41:200-209. <a href="http://ije.oxfordjournals.org/content/41/1/200.short">[link]</a></li>
<li>Leek JT, Johnson WE, Parker HS, Jaffe AE, and Storey JD. (2012) &quot;The sva package for removing batch effects and other unwanted variation in high-throughput experiments.&quot; Bioinformatics DOI:10.1093/bioinformatics/bts034 <a href="http://bioinformatics.oxfordjournals.org/content/28/6/882.long">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/sva-appnote.pdf">[pdf]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/sva-appnote-supp.pdf">[supplemental materials]</a> <a href="http://bioconductor.org/packages/release/bioc/html/sva.html">[software]</a></li>
<li>Jaffe AE, Feinberg AP, Irizarry IR, and Leek JT.(2011) &quot;Significance analysis and statistical dissection of variably methylated regions&quot; Biostatistics 13: 166-178 <a href="http://biostatistics.oxfordjournals.org/content/13/1/166.short">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/jaffe-vmrs.pdf">[pdf]</a></li>
</ul>

<h3>2011</h3>

<ul>
<li>Frazee AC, Langmead B, and Leek JT. (2011) &quot;ReCount: A multi-experiment resource of analysis-ready RNA-seq gene count datasets&quot; BMC Bioinformatics 12:449 <a href="http://www.biomedcentral.com/1471-2105/12/449">[link]</a> <a href="http://bowtie-bio.sourceforge.net/recount/">[data]</a></li>
<li>Leek JT, Taub MA, Pineda FJ. (2011) &quot;Cooperation between referees and authors increases peer review accuracy&quot; PLoS One 6:e26895 <a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0026895">[link]</a> <a href="http://biostat.jhsph.edu/%7Ejleek/peerreview/">[data]</a></li>
<li>Colantuoni C, Lipska BK, Hyde TM, Tao R, Leek JT , Colantuoni EA, Elkahlhoun AG, Herman MM, Weinberger DR, Kleinman JE.(2011) &quot;Temporal dynamics and genetic control of transcription in the human prefrontal cortex&quot; Nature 478: 519-523. <a href="http://www.nature.com/nature/journal/v478/n7370/full/nature10524.html">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/carlosbrain.pdf">[pdf]</a> <a href="http://braincloud.jhmi.edu/">[data]</a></li>
<li>Desai KH, Tan CS, Leek JT , Maier RV, Tompkins RG, Storey JD, and IHGRI.(2011) &quot;Dissecting inflammatory complications in critically injured patients by within-patient gene expression changes: A longitudinal clinical genomics study&quot; PLoS Medicine 8:e1001093 <a href="http://www.plosmedicine.org/article/info%3Adoi%2F10.1371%2Fjournal.pmed.1001093">[link]</a> <a href="http://genomine.org/trauma/">[data]</a></li>
<li>Hansen KD, Wu Z, Irizarry RI, and Leek JT. (2011) Sequencing technology does not eliminate biological variability. Nature Biotechnology, 29:572â€“573 <a href="http://www.nature.com/nbt/journal/v29/n7/full/nbt.1910.html">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/seqvar.pdf">[pdf]</a></li>
<li>Leek JT. , Storey JD (2011) &quot;The joint null criterion for multiple hypothesis tests.&quot; Statistical Applications in Genetics and Molecular Biology 10:28  <a href="http://www.degruyter.com/view/j/sagmb">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/jointnull.pdf">[pdf]</a> <a href="http://www.bioconductor.org/packages/devel/bioc/html/dks.html">[code]</a></li>
<li>Woo S, Leek JT , Storey JD (2011) &quot;A computationally efficient modular optimal discovery procedure.&quot; Bioinformatics 27:509-515 <a href="http://bioinformatics.oxfordjournals.org/content/27/4/509">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/odp-computation.pdf">[pdf]</a></li>
<li>Leek JT. (2011) &quot;Asymptotic conditional singular value decomposition for high-dimensional genomic data.&quot; Biometrics 67:344-352 <a href="http://www.ncbi.nlm.nih.gov/pubmed/20560929">[pubmed]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/acsvd.pdf">[pdf]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/code/dimensionfunc.R">[code]</a></li>
</ul>

<h3>2010</h3>

<ul>
<li>Leek JT , Scharpf RB, Corrada Bravo H * , Simcha D * , Langmead B, Johnson WE, Geman D, Baggerly K, Irizarry RA (2010), &quot;Tackling the widespread and critical impact of batch effects in high-throughput data.&quot; Nature Reviews Genetics 11:733-739 <a href="http://www.nature.com/nrg/journal/v11/n10/abs/nrg2825.html">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/batch-review.pdf">[pdf]</a> <a href="http://rafalab.jhsph.edu/batch/">[data]</a></li>
<li>Langmead B, Hansen KD*, Leek JT+ (2010), &quot;Cloud-scale RNA-sequencing differential expression analysis with Myrna.&quot; Genome Biology 11:R83. <a href="http://genomebiology.com/2010/11/8/R83/abstract">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/myrna.pdf">[pdf]</a></li>
<li>Boca SM * , Corrada Bravo H * , Brian Caffo, Leek JT+, and Parmigiani G. (2010) &quot;A decision-theory approach to interpretable set analysis for high-dimensional data.&quot; JHU Biostat Working Paper 211. <a href="http://biostats.bepress.com/jhubiostat/paper211/">[link]</a></li>
<li>Lu R, Markowetz F, Unwin RD, Leek JT , Airoldi EM, MacArthur BD, Lachmann A, Rozov R, Ma&#39;ayan A, Boyer LA, Troyanskaya OG, Whetton AD, and Lemischka IR. (2010) &quot;Systems-level dynamic analyses of fate change in murine embryonic stem cells.&quot; Nature 462: 358:362. <a href="http://www.nature.com/nature/journal/v462/n7271/full/nature08575.html">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/systems.pdf">[pdf]</a></li>
</ul>

<h3>2009 and Earlier</h3>

<ul>
<li>Leek JT. (2009) &quot;The tspair package for finding top scoring pair classifiers in R.&quot; Bioinformatics 25:1203-1204. <a href="http://bioinformatics.oxfordjournals.org/content/25/9/1203.long">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/tspair.pdf">[pdf]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/tspair/index.html">[software]</a></li>
<li>Leek JT and Storey JD. (2008) &quot;A general framework for multiple testing dependence.&quot; Proceedings of the National Academy of Sciences , 105: 18718-18723. <a href="http://www.pnas.org/content/105/48/18718.long">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/framework.pdf">[pdf]</a> <a href="http://bioconductor.org/packages/release/bioc/html/sva.html">[software]</a></li>
<li>Leek JT and Storey JD. (2007) &quot;Capturing heterogeneity in gene expression studies by `Surrogate Variable Analysis&#39;.&quot; PLoS Genetics, 3: e161. <a href="http://dx.plos.org/10.1371/journal.pgen.0030161">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/sva.pdf">[pdf]</a> <a href="http://bioconductor.org/packages/release/bioc/html/sva.html">[software]</a></li>
<li>Akey JM, Biswas S, Leek JT, and Storey JD. (2007) &quot;On the design and analysis of expression studies in human populations.&quot; Nature Genetics, 39: 807-808. <a href="http://www.nature.com/ng/journal/v39/n7/full/ng0707-807.html">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/design.pdf">[pdf]</a></li>
<li>Storey JD, Dai JY, and Leek JT. (2006), &quot;The optimal discovery procedure for large-scale significance testing, with applications to comparative microarray experiments.&quot; Biostatistics, 8: 414-432. <a href="http://biostatistics.oxfordjournals.org/content/8/2/414.long">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/odp.pdf">[pdf]</a> <a href="http://www.genomine.org/edge/">[software]</a></li>
<li>Leek JT, Monsen E, Dabney AR, and Storey JD. (2006), &quot;EDGE: Extraction and analysis of differential gene expression.&quot; Bioinformatics, 22: 507-508. <a href="http://bioinformatics.oxfordjournals.org/content/22/4/507.long">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/edge.pdf">[pdf]</a> <a href="http://www.genomine.org/edge/">[software]</a></li>
<li>Storey JD, Xiao W, Leek JT, Tompkins RG, and Davis RW. (2005), &quot;Significance analysis of timecourse microarray experiments.&quot; Proceedings of the National Academy of Sciences, 102: 12837-12842. <a href="http://www.pnas.org/content/102/36/12837.long">[link]</a> <a href="http://www.biostat.jhsph.edu/%7Ejleek/papers/time.pdf">[pdf]</a> <a href="http://www.genomine.org/edge/">[software]</a></li>
</ul>

</div>

    </div>

  </body>
</html>
